### ISSUE TITLE
Minor metric discrepancies

### ISSUE THREAD

**Issue by GabrielMiceli** [niteeshy/ar-one-sans#14]
There are several minor cosmetic issues that I've found while playing around with the static "AR One Sans Retina" font files provided in this repository.

- [x] The dot in `zero` is offset to the right by 4 units in all weights.
![zero-dot.png](https://github.com/niteeshy/ar-one-sans/assets/30030529/61c7af54-599b-4ed8-bdfb-7e2f4eb14015)

- [x] Glyph `zero.ss01` has kerning information associated with it. This information should be removed.
![zerokern](https://github.com/niteeshy/ar-one-sans/assets/30030529/be082b63-15db-4c58-8cfe-cbf770bc499a)

- [x] #15
![sevenwid](https://github.com/niteeshy/ar-one-sans/assets/30030529/db1952cd-df47-42d7-98cb-e149d6afd753)

- [x] The dot above glyphs `i` and `j` differ from the dot above glyphs `iogonek` and `i.loclTRK`.
![i-j-dot](https://github.com/niteeshy/ar-one-sans/assets/30030529/2164c6f4-4266-4e84-aee9-ae199173f604)
I'd also like to suggest aligning the dots as if the extra strokes on the left aren't there because the dots in their current state make the glyphs look unbalanced to me, especially when the font is used at smaller sizes.
![tinyidot](https://github.com/niteeshy/ar-one-sans/assets/30030529/00397aa3-9214-4401-90f5-a19871f2b238)

- [x] The `daggerdbl` glyph has non-retina top and bottom ends.
![dagger](https://github.com/niteeshy/ar-one-sans/assets/30030529/3320705f-5e13-4dcd-8207-499e8d4b0645)

**Comment by niteeshy** [comment]
Thanks for reporting this @GabrielMiceli, I will look into it and update the files soon.

**Comment by GabrielMiceli** [comment]
I decided to update this issue to cover more minor details that I found have problems in this font.

I know that a lot of it sounds nitpicky, and I recognise that many people wouldn't even notice these little details, but there are some exceptions, including me, and I don't quite feel comfortable using it knowing about some of these imperfections. 

I really would like to fix these bugs myself because I actually really like this font—it's right up there in my short list of favourite fonts, and I want to make it my new UI font—and I want it to be the best that it can be, but what truly sucks is that I'm unable to make proper contributions because this font was created in Glyphs 3, a program developed exclusively for Mac OS. The problem is that I don't own, can't afford, and am unwilling to purchase a Macintosh computer to do so for many reasons.
